### PR TITLE
Hamburger-menu

### PR DESCRIPTION
**Pull request by UmerZia**
### In this brach I:
- add a hamburger menu
- add move to selected area

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -318,22 +346,35 @@ <h3>Skills</h3>
...
           <li>
-            <a href="#"><img class="social" src="images/GitHub.svg" alt="GitHub" /></a>
+            <a href="#"
+              ><img class="social" src="images/GitHub.svg" alt="GitHub"
+            /></a>
           </li>
           <li>
-            <a href="#"><img class="social" src="images/Linkdin.svg" alt="Linkdin" /></a>
+            <a href="#"
+              ><img class="social" src="images/Linkdin.svg" alt="Linkdin"
```
Comment (1GORDON) [inline index.html:355]: Great job @UmerZia 🤝

- [ ] You may kindly make sure these social icons don't appear on top of the mobile menu as you scroll down through the page.
overlap 2

File: index.html [hunk 2]
```diff
@@ -192,7 +220,7 @@ <h2 class="work-card-h2">Profesional Art Printing Data</h2>
     </section>
 
     <!-- About Me -->
-    <section class="about-section">
+    <section id="about-me" class="about-section">
       <div class="about-div">
         <h1>About me</h1>
         <p>
```
Comment (1GORDON) [inline index.html:226]: Great design @UmerZia 🤝

- [ ] You kindly make this diamond not to appear as you scroll down through the mobile menu
overlap1

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @UmerZia ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull Request ✔️
- Linters are passing ✔️
- Professional Readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (UmerZia) [reply index.html:226]: I fixed this issue.

**Comment by UmerZia** [comment]
I fixed the mention issue by increasing z-index value of navbar.

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @UmerZia               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes requested by the previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**